src/dirs.proj

```diff
@@ -39,4 +39,10 @@
     </TraversalBuildDependsOn>
   </PropertyGroup>
 
+  <!-- Generate the shared library for running ILC tests for configuration Release-x64 -->
```

Comment:
the generation of the shared library requires the mscorlib shim to be includded in the compilation, which builds after src.builds, which is why I moved this here.